api/camunda/camunda-openapi.yaml

```diff
@@ -731,52 +837,67 @@ paths:
           description: >
             The decision requirements search query failed.
             More details are provided in the response body.
-  /authorizations:
-    post:
+
+  /authorizations/{ownerKey}:
+    patch:
       tags:
         - Authorization
-      summary: Create an authorization
-      operationId: createAuthorization
+      summary: Patch an authorization
```

Comment:
Per [this review](https://github.com/camunda/camunda/pull/22002#discussion_r1747273859) from @mesellings, this also should change to `Patch authorization` (no `an`).

Reply:
https://github.com/camunda/camunda/pull/22101